### PR TITLE
Extracting the entire message from the metrics and just store the 'alert' value

### PR DESCRIPTION
**Pull request by matzew**
Done for https://issues.jboss.org/browse/AGPUSH-1257

Should be back ported to 1.0.x branch as well

### CODE REVIEW

**Comment by matzew** [comment]
@lfryc @sebastienblanc @abstractj please review

File: admin-ui/app/views/notification.html [hunk 1]
```diff
@@ -54,7 +54,7 @@
                     </tr>
                     <tr>
                       <td>Message:</td>
-                      <td colspan="2"><strong>{ "message" : "{{ activityCtrl.parse(metric).message.alert }}" }</strong></td>
+                      <td colspan="2"><strong>{ "message" : "{{ activityCtrl.parse(metric).alert }}" }</strong></td>
```
Comment (lfryc) [inline admin-ui/app/views/notification.html]: I wonder whether it make sense to wrap the alert in `{ "message" : "...alert..."}` when it is not actually the message format (in fact it is not used anywhere in this form afaik).

It appears in the notification detail as:

````
Request IP:	127.0.0.1	Full Request
Message:	{ "message" : "test" }
````

Could we state just: `Message: {{ activityCtrl.parse(metric).alert }}`

**Comment by lfryc** [comment]
Similarly if I click on Full request, I get:

````
{
    "ipAddress": "127.0.0.1",
    "clientIdentifier": "AeroGear UnifiedPush Console",
    "alert": "test",
    "criteria": {
        "categories": null,
        "variants": null,
        "alias": null,
        "deviceType": null
    },
    "config": {
        "ttl": -1
    }
}
````

For consistency with the actual push message format, it could be:

````
{
    "ipAddress": "127.0.0.1",
    "clientIdentifier": "AeroGear UnifiedPush Console",
    "message": {
        "alert": "test"
    },
    "criteria": {
        "categories": null,
        "variants": null,
        "alias": null,
        "deviceType": null
    },
    "config": {
        "ttl": -1
    }
}
````

Notice the nested `message`

Shouldn't we state somewhere that the message was stripped?

**Comment by matzew** [comment]
@lfryc did the 
```
 <td>Message:</td>
                      <td colspan="2"><strong>{{ activityCtrl.parse(metric).alert }}</strong></td>
```

**Comment by matzew** [comment]
@lfryc regarding:
```
"message": {
        "alert": "test"
    },
```

I can do that, but since we really store the alert I saw zero sense in actually prepending that other stuff might be stored as well (user will get confiused, that no other properties are listed)

> Shouldn't we state somewhere that the message was stripped (that does not play nice with "Full Request" label)?

The doc will say "what's stored and what not". There is a JIRA for that.

Full request may not make too much sense, but it shows a bit more (e.g. all metrics). I am fine in either keeping foll request, or renaming to `Details`

**Comment by lfryc** [comment]
> I can do that, but since we really store the alert I saw zero sense in actually prepending that other stuff might be stored as well (user will get confiused, that no other properties are listed)

If we are not trying to mimic push message format anyhow, then this is fine.

+1 to `Details` - sounds great

**Comment by matzew** [comment]
@lfryc cool - Details it is :-)

and I rebased it

**Comment by lfryc** [comment]
@matzew yea, just may be one more update - now the link is `Details`, but the header of the modal panel is `Full Request`

the update here is required: https://github.com/aerogear/aerogear-unifiedpush-server/blob/master/admin-ui/app/views/dialogs/request.html#L6

**Comment by matzew** [comment]
@lfryc changed it to Detaiils... :)

**Comment by matzew** [comment]
well, Request Detauls it is ...

**Comment by lfryc** [comment]
:+1: thanks!

**Comment by sebastienblanc** [comment]
:+1: looks good to me

**Comment by matzew** [comment]
rebased

**Comment by matzew** [comment]
ported to 1.0.x f7144e950ca5013a240fa34979e35eaf68eb6374

**Comment by abstractj** [comment]
@matzew the had only 6 hours and was merged already? Not sure why I was asked to review

**Comment by matzew** [comment]
PR was submitted on Wednesday, sorry for being a bit fast here.

Want me to revert?

On Thursday, February 5, 2015, Bruno Oliveira <notifications@github.com>
wrote:

> @matzew <https://github.com/matzew> the had only 6 hours and was merged
> already? Not sure why I was asked to review
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/488#issuecomment-73116288>
> .
>


-- 
Sent from Gmail Mobile

**Comment by abstractj** [comment]
@matzew 

And nope, I don't.

**Comment by matzew** [comment]
that was the merge. I am sorry, if that was merged to.

But the original PR was submitted on Wednesday, that's when I added the comment about for wich JIRA it was done:
https://www.dropbox.com/s/azlcy13o61dett1/Screen%20Shot%202015-02-06%20at%2012.41.21.png?dl=0

But sure if that brings us anything here - fact is, I merged it perhaps to early. I am sorry for that

**Comment by abstractj** [comment]
@matzew not a big deal, all good

**Pull request merged**